src/components/Hero/_Hero.scss

```diff
@@ -57,6 +57,7 @@
 
         .hero__heading {
             @include space-stack-none;
+            margin-top: 0;
```

Comment:
Doesn't look like `.hero__heading` is applying any top margin, so I'm not convinced DS CSS is causing the discrepancy—I might take a look in SFR, see what's happening exactly 🤔

Reply:
I think after some investigation this is happening in the `dgx-footer`. Can you remove this line and override it in SFR?